### ISSUE TITLE
[Feature Request] Improved curtains

### ISSUE THREAD

**Issue by Jay420blaze** [crispytwig/Another-Furniture-Mod#68]
Hey, overall this is my favorite furniture mod with features i've wanted from a furniture mod such as tucking in chairs. I think all of the furniture is very well made and works in all style of interiors. However, one issue I'm currently facing is with curtains only having a "max length" of 2 blocks. This makes it so even on slightly bigger windows curtains can not cover a full window. and such look a little weird.

Examples:

**Small windows (3x2)** - ✔️

![2024-10-01_17 41 41](https://github.com/user-attachments/assets/ad8744e6-3cc2-4432-99aa-bb22ca515927)

![2024-10-01_17 41 46](https://github.com/user-attachments/assets/289c70bb-8f76-4e2c-bec8-2a73c807182a)

**big window (4x anything bigger then 2)** - ❌

![2024-10-01_17 40 09](https://github.com/user-attachments/assets/82785931-c83b-4d8d-a3ca-8458da87fc6d)

![2024-10-01_17 41 20](https://github.com/user-attachments/assets/b2f98e7b-6187-44c0-a78f-4672d69a9ae8)

Tl;dr:
Curtains can only be 2 blocks "long" thus bigger windows can not be fully covered by curtains, I would love to see the ability to make curtains "longer" so they can be used on windows of any size.

**Comment by xhatz** [comment]
Please YES

**Comment by Beepa11** [comment]
YES I PRAY

**Comment by BrainDifff** [comment]
would be sick

**Comment by Ghannkaru** [comment]
Yes, please add this feature! The screenshots on the Modrinth project page show something like this, so was it in an older version?
